docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -2,234 +2,205 @@
+[role="screenshot"]
+image::images/rule-details-alerts-inactive.png[Index Threshold chart]
 
-Rules and actions run as background tasks by each {kib} instance at a default rate of ten tasks every three seconds.
+The end date is related to the alert interval (IIRC, 30 “intervals” worth of time). You can use this view to see if the rule is getting the data you expect, and visually compare to the threshold value (a horizontal line in the graph). If the graph does not contain any lines at all except for the threshold line, then the rule is having an issue of some kind. Either there is no data available given the index and fields specified, a permission error, etc.
+Diagnosing these may be difficult - but there may be log messages for error conditions. 
 
-If many rules or actions are scheduled to run at the same time, pending tasks will queue in {es}. Each {kib} instance then polls for pending tasks at a rate of up to ten tasks at a time, at three second intervals. Because rules and actions are backed by tasks, it is possible for pending tasks in the queue to exceed this capacity and run late.
+[float]
+[[alerting-rest-api]]
+=== Using REST APIs
 
-For details on diagnosing the underlying causes of such delays, see <<task-manager-health-tasks-run-late>>.
+There is a fairly rich set of HTTP endpoints to introspect and manage rules and connectors.  
+In addition, there is a command-line client that uses legacy Rules and Connectors APIs, which be easier to to use, but it needs to be updated for the new APIs:
 
-Alerting and action tasks are identified by their type.
+https://github.com/pmuellr/kbn-action
+executing an action ad-hoc
+One of the http endpoints available for actions is the POST _execute API (reference).  You can use this to “test” an action.  For instance, if you have a server log action created, you can execute it via curling the endpoint:
+[source, txt]
+--------------------------------------------------
+curl -X POST -k \
+ -H 'kbn-xsrf: foo' \
+ -H 'content-type: application/json' \
+ https://elastic:changeme@localhost:5601/api/actions/connector/a692dc89-15b9-4a3c-9e47-9fb6872e49ce/_execute \
+ -d '{"params":{"subject":"hallo","message":"hallo!","to":["me@example.com"]}}'
+--------------------------------------------------
 
-* Alerting tasks always begin with `alerting:`. For example, the `alerting:.index-threshold` tasks back the <<rule-type-index-threshold, index threshold stack rule>>.
-* Action tasks always begin with `actions:`. For example, the `actions:.index` tasks back the <<index-action-type, index action>>.
+Another option is CLI tools to list / create / edit / delete alerts and actions are available in https://github.com/pmuellr/kbn-action[kbn-action]. Install via command:
+[source, txt]
+--------------------------------------------------
+    npm install -g pmuellr/kbn-action  
```

Comment:
I don't think pmuellr is right here.

Reply:
This repository will be moved from the pmuellr own repository to Kibana - open [issue](https://github.com/elastic/kibana/issues/101503)